research/picai/preprocessing_transforms.py

```diff
@@ -177,11 +178,12 @@ def crop_or_pad(
             idx_end = idx_start + size[i]
             slicer[i] = slice(idx_start, idx_end)
 
+    new_image = image[tuple(slicer)]
+
     # crop and/or pad image
-    if isinstance(image, sitk.Image):
+    if any(pad[0] > 0 or pad[1] > 0 for pad in padding):
```

Comment:
Any reason we're "dropping" support for non-sitk.Image types?

Reply:
I definitely want to support a wider variety of image types and for some its pretty trivial, but at this point just to ensure everything matches how its currently typed which doesn't include numpy arrays

Reply:
Sounds good. I'm good with doing it. Was just curious.